cpp-linux/apt/build.sh

```diff
@@ -48,7 +49,11 @@ run cp /host/tmp/${PACKAGE}-${VERSION}.tar.gz \
 run cd build
 run tar xfz ${PACKAGE}_${VERSION}.orig.tar.gz
 run cd ${PACKAGE}-${VERSION}/
-run cp -rp /host/tmp/debian debian
+if [ "$distribution" = "ubuntu" ] && [ "$release" = "14.04" ]; then
+  run cp -rp /host/tmp/debian.ubuntu-14.04 debian
+else
+  run cp -rp /host/tmp/debian debian
+fi
```

Comment:
Ah, we need to use "release" instead of "code_name" to use "ubuntu-14.04"...

OK. Please rename "ubuntu-${XX}.${YY}" to "ubuntu-${CODE_NAME}" like "ubuntu-trusty" and change the code like the followings:

```sh
specific_debian_dir="debian.${distribution}-${code_name}"
if [ -d "${specific_debian_dir}" ]; then
  run cp -rp "/home/tmp/${specific_debian_dir}" debian
else
  run cp -rp "/home/tmp/debian" debian
fi
```

It's more general. We don't need to change `build.sh` when we add other `debian.XXX` directory.